team.tex

```diff
@@ -0,0 +1,123 @@
+significant software in Telescope \& Site and Camera. The project manager delegates
+responsibility Deputy PM is responsible for the smooth running of software in all
+subsystems.
+\item Ultimately, the DMPM, in conjunction with his/her peer Project Managers (Telescope,
+Camera), is responsible for delivering an integrated Rubin system.
+\item The Deputy Project Manager has the responsibility for resolving conflicts involving IT
+and Software between IT, T\&S, Camera and DM.
+\item The Deputy PM as a manager of mangers is responsible for obtaining and maintaining
+managers in senior software management roles. Specifically head of Telescope and
+Site Software, IT Chile IT Tucson and Data Management.
+\item Assist software managers in definition of scope, funding requests, risk management
+and mitigation.
+\item Ensure compatible and where possible similar work and management practices for
+software across subsystems.
+\item Assist with reviews and review preparations from a software perspective.
+\item Provide leadership to senior managers involved in IT and Software across Rubin.
+\item Foster a good, equitable, diverse and inclusive working environment within Rubin.
+\end{itemize}
+\item Authorities
+\begin{itemize}
+\item  In the absence of the PM the deputy carries full authority and decision making powers
+of the PM.
+\item Approve or execute as appropriate all Software/IT outsourcing contracts.
+\item Approve or execute as appropriate all IT equipment and licensing purchases.
+\end{itemize}
+\item Accountabilities
+\begin{itemize}
+\item Budgeting of software and IT across Rubin should be monitored.
+\item Decisions made on software and IT especially across subsystem boundaries.
+\item Actions and decisions of senior software and IT managers across Rubin Specifically head of Telescope and Site Software, IT Chile IT Tucson and Data Management.
+
+\end{itemize}
+\end{itemize}
+
+
+\subsubsection{Operations Software} \label{sec:opssw}
+
+In Rubin Operations the  head of software is under  discussion.
+Trimming down the construction role we might consider:
+
+\begin{itemize}
+\item Roles
+\begin{itemize}
+\item Facilitate all software decision making across the project.
+\item Facilitate all IT decisions across the project.
+\item Manager of managers for software and IT roles – even if such managers are in different
+departments.
+\item Mentor senior IT and Software personnel.
+\end{itemize}
+Responsibilities
+\begin{itemize}
+\item Responsible for the smooth running of software in all subsystems.
+\item Responsible for maintenance of the over all Rubin software system in conjunction  with the various software leads/managers.
+\item Responsible for resolving conflicts involving IT and Software between IT, T\&S, Camera, EPO  and DM.
+\item Responsible for obtaining and maintaining managers in senior software management roles. Specifically head of Telescope and Site Software, DevOps Chile, IT Tucson and Data Management.
```

Comment:
same Telescope and Site becomes ROO

Reply:
its the software lead though not ROO lead .. i put some words